### DISCUSSION TITLE
MarginalLogLikelihood v/s LogMarginalLikelihood

### DISCUSSION THREAD

**Discussion by patel-zeel** [cornellius-gp/gpytorch#1805] [Q&A] [answered]
In the early stage of learning GPs, I was always confused between likelihood and marginal likelihood. For a certain period, I was thinking that training GPs is MLE because at some places I found people using term likelihood as a proxy for marginal likelihood. To make the difference very clear, isn't LogMarginalLikelihood more explicit than MarginalLogLikelihood? [[first line of the answer](https://stats.stackexchange.com/a/280109/305392)]

**Comment by gpleiss** [comment, marked as answer]
> To make the difference very clear, isn't LogMarginalLikelihood more explicit than MarginalLogLikelihood?

Yes, I do think this is the correct name. Changing things in GPyTorch would be possible, but would require appropriately deprecating the current naming convention.

**Reply by patel-zeel** [reply]
Thank you for the answer, @gpleiss. So, how would you suggest going about it?
